core/src/main/java/com/radixdlt/rev2/REv2TransactionsAndProofReader.java

```diff
@@ -101,21 +90,12 @@ public REv2TransactionsAndProofReader(REv2TransactionAndProofStore transactionSt
   public LedgerExtension getTransactions(DtoLedgerProof start) {
     final var startStateVersionInclusive = start.getLedgerHeader().getStateVersion() + 1;
 
-    final var rawTxnsAndProofOpt =
-        transactionStore.getTxnsAndProof(
-            startStateVersionInclusive,
-            MAX_TXNS_FOR_RESPONSES_SPANNING_MORE_THAN_ONE_PROOF,
-            MAX_TXN_BYTES_FOR_A_SINGLE_RESPONSE);
+    final var rawTxnsAndProofOpt = transactionStore.getTxnsAndProof(startStateVersionInclusive);
 
     if (rawTxnsAndProofOpt.isEmpty()) {
       log.error(
-          "Impossible to build a chain of transactions from state version "
-              + startStateVersionInclusive
-              + " that ends with a proof that is: under "
-              + MAX_TXNS_FOR_RESPONSES_SPANNING_MORE_THAN_ONE_PROOF
-              + " transactions and under "
-              + MAX_TXN_BYTES_FOR_A_SINGLE_RESPONSE
-              + " total payload size!");
+          "Impossible to build a chain of txns from state version {} fitting within the limits",
```

Comment:
yeah the problem was only specific to Core API layer, `handle_stream_transactions()`, and only because it wanted to return `previous_state_identifiers` (see https://rdxworks.slack.com/archives/C01M90Y2448/p1695389384296069) and it got fixed in https://github.com/radixdlt/babylon-node/pull/688 and now is covered by an int-test `requesting_state_version_out_of_bounds_returns_error()`.

The `REv2TransactionsAndProofReader`'s method (and its rust impl `get_txns_and_proof()`) was always resilient to "get far future" (an empty proofs iter results in `return None;`) and I haven't touched it (only adjusted the error message).